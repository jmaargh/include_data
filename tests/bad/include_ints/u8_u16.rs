use include_data::include_ints;

fn main() {}

// Type mismatch
static BYTES_32_u8_u16: &[u8] = include_ints!(u16, "../../test_data/binary_32");
