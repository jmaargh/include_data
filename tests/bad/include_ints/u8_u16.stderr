error[E0308]: mismatched types
 --> tests/bad/include_ints/u8_u16.rs:6:33
  |
6 | static BYTES_32_u8_u16: &[u8] = include_ints!(u16, "../../test_data/binary_32");
  |                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `&[u8]`, found `&[u16]`
  |
  = note: expected reference `&'static [u8]`
             found reference `&'static [u16]`
  = note: this error originates in the macro `include_ints` (in Nightly builds, run with -Z macro-backtrace for more info)
